tests/integration/crunchers/test_cruncher_quantile_rolling_windows.py

```diff
@@ -187,8 +187,12 @@ def test_relationship_usage(self, simple_df, use_ratio, caplog):
     @pytest.mark.parametrize("use_ratio", [True, False])
     def test_numerical_relationship(self, use_ratio):
         # Calculate the values using the cruncher for a fairly detailed dataset
-        large_db = IamDataFrame(self.large_db.copy())
-        tcruncher = self.tclass(large_db)
+        larger_db = IamDataFrame(self.large_db)
+        larger_db["model"] = larger_db["model"] + "_2"
+        larger_db["value"] = larger_db["value"] - 0.5
+        larger_db.append(IamDataFrame(self.large_db), inplace=True)
+        larger_db = IamDataFrame(larger_db.data)
```

Comment:
I find that when I act directly on the columns of data the dataframe (lines 191-192) does not always update properly, so I usually refresh it in this way. It might work without it, I don't even try any more.

Reply:
yeah, it's needed

Reply:
if you have the energy, would be worth raising an issue in pyam about this